### DISCUSSION TITLE
What are the reasons for the v5.0.0-rc3 release candidate?

### DISCUSSION THREAD

**Discussion by strzinek** [containers/podman#21798] [General]
May I ask why you are releasing a release candidate when the description implies that another release candidate is certain to follow? You could have saved yourself the expense of releasing it if you knew in advance that it could not be a release. Or you could have called it a test release.

**Comment by Luap99** [comment]
A release candidate is always a test release.

**Reply by baude** [reply]
In addition to what @Luap99 says, we have been asked by the community, community distributions, and enterprise distributions to do a series of rc's before every release.  like @Luap99 says, this allows people to test code that is not likely to change significantly as a representative of the release.  moreover, distributions like to use the rc's to ensure their packaging (and dependencies) and builds are going to be smooth once the release is actually complete.
